docs/src/styles/docs/main.scss

```diff
@@ -19,16 +19,17 @@
 
 .docs-content-body {
   padding: var(--amplify-space-xxl);
-  
-  & table {
+
+  & table:not(.amplify-table) {
```

Comment:
:not is needed here because with this update we are making all of our default styling more generic and here is a spot where our default styling for our table component was more specific than this rule but after the update it becomes less specific.